intg-appletv/tv.py

```diff
@@ -159,6 +161,9 @@ def __init__(
         self._poll_interval: int = 10
         self._state: DeviceState | None = None
         self._app_list: dict[str, str] = {}
+        self._available_output_devices: dict[str, str] = {}
+        self._output_devices: OrderedDict[str, [str]] = OrderedDict()
+        self._output_device: list[OutputDevice] = []
```

Comment:
This is correct, _output_device is the selected combination of multiple devices to stream to.
I think this is a temporary approach, on discord you said that you would think after a more flexible solution (the ability to select one or more speakers instead of having one selection of multiple combinations).
I understand your suggestion : the combination could change outside remote use. I'll try to think about something better

Reply:
This approach is fine for now, I only suggest a naming change, since it's not a single output device but a list of devices. Since the plural name is already taken for the OrderedDict, that one should be renamed to.

Reply:
Actually I just removed the var ```_output_device``` which was useless. However the ```_output_devices``` is necessary to hold a map of possible combinations after the network scan (which is done only at connection initialization, although the list can change after)